docs/workloads/azure/data/intro_data_azure.md

```diff
@@ -0,0 +1,48 @@
+---
+id: intro_data_azure
+title: Architecture Overview
+sidebar_label: Architecture Overview
+hide_title: true
+hide_table_of_contents: false
+description: Azure Data Platform - Architecture Overview
+keywords:
+  - data
+  - python
+  - etl
+  - databricks
+  - azure
+  - adf
+---
+
+# Stacks – Azure Data Platform
+
+## Overview
+
+Housing an Azure data platform template, [stacks-azure-data](https://github.com/amido/stacks-azure-data)
+repository leverages **Azure Data Factory** for data ingestion and orchestration of data processing
+using **Databricks**. It also employs **Azure Data Lake Storage Gen2** for data lake storage.
+The solution's data workload naming convention originates from Databricks' Medallion Architecture,
+a system emphasizing structured data transformation layers. Key elements of the solution include:
```

Comment:
We can match the wording in the main README if we're happy with it.

Reply:
done